src/Tree.tsx

```diff
@@ -516,7 +516,8 @@ class Tree extends React.Component<TreeProps, TreeState> {
     }
 
     // Skip if drag node is self
-    if (dragNode.props.eventKey === dropTargetKey && dropLevelOffset === 0) {
+    // Remember to check dragNode exists - it could be undefied if iot is from another tree
```

Comment:
```suggestion
    // Remember to check dragNode exists - it could be undefined if iot is from another tree
```

Reply:
By the way what is `iot`?

Reply:
sorry - it is a typo. Should be  `it could be undefined if it is from another tree`